### ISSUE TITLE
Can we remove restriction that imports come before everything else?

### ISSUE THREAD

**Issue by jclark** [ballerina-platform/ballerina-spec#662]
Can we remove the imports-must-come-first-before-all decls requirement? The primary need is for imports to come before its used, beyond that why do we care? For code styling its best to put them at the top for sure, but is it really beneficial to require that at a language level?

_Originally posted by @sanjiva in https://github.com/ballerina-platform/ballerina-spec/issues/660#issuecomment-735049254_

**Comment by jclark** [comment]
This is not a new requirement. It's been like this, since I was involved with Ballerina. You have on numerous occasions advocated the principle that Ballerina should not need coding standards: things that would be left to coding standards in other languages would be enforced syntactically in Ballerina.

It's not a principle I like, so I am OK to relax it. However, I think the import of a  module prefix should occur before the first use of the  module prefix.

**Comment by sanjiva** [comment]
Yes I was for having a single way to do things as a way to make it easy for Mort. However, we've gone past that now so I don't think that should get in the way any more. The IDE can still put them at the top as that's what most people are used to.

+1 of course for import before use.
